test/utils/mutations/remove-mutation.ts

```diff
@@ -46,3 +48,18 @@ export const removeUserFromGroupVariablesData = (
   const responseData = JSON.stringify(variables);
   return responseData;
 };
+
+// Execute function
+/**
+ * Removes user from an entity
+ * @param mut name of remove mutation
+ * @param variable name of function containing mutation vriables
+ * @param role role type
+ */
+export const removeUserFromEntity = async (
```

Comment:
Actually, it just executes a given mutation. There are no criteria if it's a delete one or not.

Reply:
Moved the function under `graphql.request.ts` file as it will be commonly used.